src/components/payment-qr/container/index.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import type { DispatchType } from 'constants/redux'
+
+import * as ReactRedux from 'react-redux'
+import * as Reselect from 'reselect'
+
+import selectors from 'redux/selectors'
+import shoppingCartAction from 'redux/actions/shopping-cart'
+
+import * as PaymentQR from '../index'
+
+const mapStateToProps = Reselect.createStructuredSelector({
+  order: selectors.orders.shoppingCart,
+  orderId: selectors.users.shoppingCartOrderId,
+  walletAddress: selectors.wallet.address,
+})
+
+const mapDispatchToProps = (dispatch: DispatchType) => ({
+  onSellMoreItems: () => {
+    dispatch(shoppingCartAction.reset())
+  },
+  onRetry: () => {
+    dispatch(shoppingCartAction.resetTransaction())
```

Comment:
I would suggest putting this method in order actions

Reply:
I'll do that if there'll be a need to reset transaction for an order that is not in the shopping cart.